industrial_robot_client/src/joint_trajectory_action.cpp

```diff
@@ -45,12 +45,28 @@ const double JointTrajectoryAction::DEFAULT_GOAL_THRESHOLD_ = 0.01;
 JointTrajectoryAction::JointTrajectoryAction() :
     action_server_(node_, "joint_trajectory_action", boost::bind(&JointTrajectoryAction::goalCB, this, _1),
                    boost::bind(&JointTrajectoryAction::cancelCB, this, _1), false), has_active_goal_(false),
-                   controller_alive_(false), has_moved_once_(false), name_("joint_trajectory_action")
+                   controller_alive_(false), has_moved_once_(false), name_("joint_trajectory_action"),
+                   ignore_motion_server_error_(false), consider_status_unknowns_ok_(false)
```

Comment:
```suggestion
                   ignore_motion_server_error_(false), consider_status_unknowns_ok_(true)
```
It's a bit too strict like that

I didn't see the initialization here on first glance, doing it here or in the header is a mater of preference I guess

Reply:
setting this to `true` does decrease the value of the checks.

Ideally this would be a per-field setting, but that was a bit of a bridge-too-far for me when I wrote this.

Reply:
Yes, it's a bit less strict that way but also less likely to actually break setups (?). Hard to know without looking up all the drivers out there